src/kpatch_user.c

```diff
@@ -882,7 +984,6 @@ execute_cmd(int argc, char *argv[])
 int main(int argc, char *argv[])
 {
 	int opt;
-
```

Comment:
Please avoid unnecessary edits. I like lines after variable declarations, let's keep this one :)